content/docs/rfcs/16/README.md

```diff
@@ -191,6 +174,117 @@ The `get_waku_v2_relay_v1_messages` method returns a list of messages that were
 
 - **`Array`[[`WakuMessage`](#wakumessage)]** - the latest `messages` on the polled `topic` or an [error](https://www.jsonrpc.org/specification#error_object) on failure.
 
+## Relay Private API
+
+The Private API provides functionality to encrypt/decrypt `WakuMessage` payloads using either symmetric or asymmetric cryptography. This allows backwards compatibility with [Waku v1 nodes](https://github.com/vacp2p/specs/blob/master/specs/waku/v1/waku-1.md).
+It is the API client's responsibility to keep track of the keys used for encrypted communication. Since keys must be cached by the client and provided to the node to encrypt/decrypt payloads, a Private API SHOULD NOT be exposed on non-local or untrusted nodes.
```

Comment:
> a Private API SHOULD NOT be exposed on [non-local] or untrusted nodes.

I find it to be an odd phrasing. How can one expose an API on an untrusted node? Would be good to rephrase.
I guess there are two warnings:
- Do not expose your private API to untrusted actors (but if the nwaku node does not hold the key then is the encryption/decryption really the reason not to do that?)
- Do not use the relay private API on a node you don't trust (they could be caching your keys)